ci-operator/step-registry/upi/libvirt/install/upi-libvirt-install-commands.sh

```diff
@@ -0,0 +1,244 @@
+#!/bin/bash
```

Comment:
This file is the core logic.
It should be named: `upi-install-libvirt-commands.sh`
I had left this placeholder for it, because it thought it would be cleaner: `upi-install-libvirt-cluster-commands.sh`